withdraw-by-check/REPWRITERSPECS/BANNO.CHECK.WITHDRAW.V1.POW

```diff
@@ -589,7 +613,13 @@ PROCEDURE VALIDATEACCOUNT
        SLDESCRIPTION=TMPCHR
 
        IF VALIDSHARETYPES(SHARE:TYPE)<>TRUE THEN
-        VALIDATESTATUS=VALIDATEINVALIDSLTYPE
+        DO
+         VALIDATESTATUS=VALIDATEINVALIDSLTYPE
+         IF SLTYPELENGTH=2 THEN
```

Comment:
Just to simplify this, you can always format to 4 digits.  But, I suppose, if you are worried about the string length you can leave this in.

Reply:
made suggested change